### ISSUE TITLE
Respect node require modules resolve mechanism

### ISSUE THREAD

**Issue by JiLiZART** [ai/size-limit#153]
When I updated size-limit and installed preset on root package.json, I still get the error.
I think because size-limit searches preset dependency using `process.cwd()` and it's not how `require` resolves modules in node. 
![image](https://user-images.githubusercontent.com/62051/72918253-06db4c00-3d4e-11ea-8644-e72fe6582a58.png)

I can't install preset to everey separate package in my lerna monorepo :(

**Comment by ai** [comment]
Can you send PR (so you will be able to test the fix?)

**Comment by JiLiZART** [comment]
What did you mean "Can you send PR"? You need example repository to reproduce this problem?

**Comment by ai** [comment]
I mean, can you send pull request with the fix?

You can fork the Size Limit, add this fork to your project (by `yarn add size-limit@../size-limit/packages/size-limit`), fix the code, commit it to branch and send pull request.

**Comment by skad0** [comment]
> What did you mean "Can you send PR"? You need example repository to reproduce this problem?

I might try to resolve an issue, but I do prefer to have a repo with reproducible scenario if possible.

**Comment by ai** [comment]
@skad0 it will be faster to create test and fix the problem (for the simple case like this), rather than creating repo to reprogram issue and then anyway create tests and fix.

**Comment by binjospookie** [comment]
Have the same problem. Looks like the problem is here: https://github.com/ai/size-limit/blob/master/packages/size-limit/run.js#L39
So we need to find correct way to `size-limit`. @ai am I right? Maybe you have an idea how to solve this?

**Comment by ai** [comment]
@binjospookie tell me more about the problem? How do you run Size Limit?

**Comment by binjospookie** [comment]
I have a monorepo (lerna). Structure looks like:
```
node_modules (hoist)
package.json
....
packages -> some packages (A, B, C)
```
Now each package (A,B,C) has `@size-limit/preset-small-lib && size-limit` and script `"size": "size-limit"` at own `package.json`. And it works fine. But very inconvenient to update with new versions of size-limit.

I just want to add `@size-limit/preset-small-lib && size-limit` into root `package.json`, but when I call `npm run size` I get an error:
```
> @package-name@1.5.4 size /some-path/packages/package-name
> size-limit

Install Size Limit preset depends on type of the project

For application, where you send JS bundle directly to users
  npm install --save-dev @size-limit/preset-app
For frameworks, components and big libraries
  npm install --save-dev @size-limit/preset-big-lib
For small (< 10 KB) libraries
  npm install --save-dev @size-limit/preset-small-lib
Check out docs for more complicated cases
  https://github.com/ai/size-limit/
```

**Comment by ai** [comment]
Can you put Size Limit config to the project root with paths like `packages/a/…`? Will it solve the problem?

**Comment by binjospookie** [comment]
Nearly. In this way Size Limit calculates all packages (A,B,C) at one time, but it's impossible to run Size Limit for package A only (for example).

**Comment by ai** [comment]
@binjospookie what use case do you have to run Size Limi only for one package? (maybe we can add API to call single test)

**Comment by binjospookie** [comment]
My  `prepublishOnly` scripts looks like `npm run build && size-limit`. All packages in repo are independent and it’s not necessary to calculate size of A,B,C for A publishing.

**Comment by ai** [comment]
@binjospookie seems like your case of config finding is a different one for this issue.

Why not use Size Limit in `test` script rather than ` prepublishOnly`? Size Limit viloation is not so critical to stop release. It is better to be called on CI on pull request rather than on publishing.

**Comment by binjospookie** [comment]
@ai yes, you're right.
I put Size Limit config to the project root.
Thank you!

**Comment by exah** [comment]
I submitted PR #211 with fix. 

@ai please, review :)

**Comment by ai** [comment]
The fix was released in 4.9.2.
